client/src/views/Profile/index.vue

```diff
@@ -49,7 +47,10 @@ export default {
 
 <style lang="scss" scoped>
   .wrapp {
-    height: 500px;
+    min-height: calc(100vh - 260px);
+    .profile-content {
+      background: url("https://img5.goodfon.ru/wallpaper/nbig/1/ab/zima-sneg-snezhinki-fon-christmas-blue-winter-background-s-7.jpg");
```

Comment:
I'd suggest getting image from free service providers like pexels.com and store it in projects assets